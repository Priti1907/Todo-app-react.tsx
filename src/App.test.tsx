import React from 'react';
import { render} from '@testing-library/react';
import App from './App';

describe('snapshot testing for App page', () => {
  it('should render', () => {
    const view = render(<App />);
    expect(view).toMatchSnapshot();
  });
});








