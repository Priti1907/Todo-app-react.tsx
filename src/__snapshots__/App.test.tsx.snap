// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div
    class="container"
  >
    <header>
      <p
        class="para"
      >
        Todos
      </p>
    </header>
    <div
      class="container-form"
    >
      <section>
        <h1>
          Sign in
        </h1>
        <div>
          <input
            class="uname"
            id="username"
            placeholder="Username"
            type="email"
            value=""
          />
        </div>
        <br />
        <div>
          <input
            class="pwd"
            id="password"
            placeholder="Password"
            type="password"
            value=""
          />
          <button
            class="btn"
          >
            Sign in
          </button>
        </div>
        <div>
          <p
            class="lnk"
          >
            <a
              class=""
              href="/register"
            >
              New here? Register now
            </a>
          </p>
        </div>
      </section>
    </div>
  </div>
</div>
`;

exports[`renders learn react link 1`] = `
<div>
  <div
    class="container"
  >
    <header>
      <p
        class="para"
      >
        Todos
      </p>
    </header>
    <div
      class="container-form"
    >
      <section>
        <h1>
          Sign in
        </h1>
        <div>
          <input
            class="uname"
            id="username"
            placeholder="Username"
            type="email"
            value=""
          />
        </div>
        <br />
        <div>
          <input
            class="pwd"
            id="password"
            placeholder="Password"
            type="password"
            value=""
          />
          <button
            class="btn"
          >
            Sign in
          </button>
        </div>
        <div>
          <p
            class="lnk"
          >
            <a
              class=""
              href="/register"
            >
              New here? Register now
            </a>
          </p>
        </div>
      </section>
    </div>
  </div>
</div>
`;

exports[`snapshot testing for A page should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="container"
        >
          <header>
            <p
              class="para"
            >
              Todos
            </p>
          </header>
          <div
            class="container-form"
          >
            <section>
              <h1>
                Sign in
              </h1>
              <div>
                <input
                  class="uname"
                  id="username"
                  placeholder="Username"
                  type="email"
                  value=""
                />
              </div>
              <br />
              <div>
                <input
                  class="pwd"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
                <button
                  class="btn"
                >
                  Sign in
                </button>
              </div>
              <div>
                <p
                  class="lnk"
                >
                  <a
                    class=""
                    href="/register"
                  >
                    New here? Register now
                  </a>
                </p>
              </div>
            </section>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="container"
      >
        <header>
          <p
            class="para"
          >
            Todos
          </p>
        </header>
        <div
          class="container-form"
        >
          <section>
            <h1>
              Sign in
            </h1>
            <div>
              <input
                class="uname"
                id="username"
                placeholder="Username"
                type="email"
                value=""
              />
            </div>
            <br />
            <div>
              <input
                class="pwd"
                id="password"
                placeholder="Password"
                type="password"
                value=""
              />
              <button
                class="btn"
              >
                Sign in
              </button>
            </div>
            <div>
              <p
                class="lnk"
              >
                <a
                  class=""
                  href="/register"
                >
                  New here? Register now
                </a>
              </p>
            </div>
          </section>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`snapshot testing for App page should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="container"
        >
          <header>
            <p
              class="para"
            >
              Todos
            </p>
          </header>
          <div
            class="container-form"
          >
            <section>
              <h1>
                Sign in
              </h1>
              <div>
                <input
                  class="uname"
                  id="username"
                  placeholder="Username"
                  type="email"
                  value=""
                />
              </div>
              <br />
              <div>
                <input
                  class="pwd"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
                <button
                  class="btn"
                >
                  Sign in
                </button>
              </div>
              <div>
                <p
                  class="lnk"
                >
                  <a
                    class=""
                    href="/register"
                  >
                    New here? Register now
                  </a>
                </p>
              </div>
            </section>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="container"
      >
        <header>
          <p
            class="para"
          >
            Todos
          </p>
        </header>
        <div
          class="container-form"
        >
          <section>
            <h1>
              Sign in
            </h1>
            <div>
              <input
                class="uname"
                id="username"
                placeholder="Username"
                type="email"
                value=""
              />
            </div>
            <br />
            <div>
              <input
                class="pwd"
                id="password"
                placeholder="Password"
                type="password"
                value=""
              />
              <button
                class="btn"
              >
                Sign in
              </button>
            </div>
            <div>
              <p
                class="lnk"
              >
                <a
                  class=""
                  href="/register"
                >
                  New here? Register now
                </a>
              </p>
            </div>
          </section>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`snapshot testing for testkits page should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="container"
        >
          <header>
            <p
              class="para"
            >
              Todos
            </p>
          </header>
          <div
            class="container-form"
          >
            <section>
              <h1>
                Sign in
              </h1>
              <div>
                <input
                  class="uname"
                  id="username"
                  placeholder="Username"
                  type="email"
                  value=""
                />
              </div>
              <br />
              <div>
                <input
                  class="pwd"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
                <button
                  class="btn"
                >
                  Sign in
                </button>
              </div>
              <div>
                <p
                  class="lnk"
                >
                  <a
                    class=""
                    href="/register"
                  >
                    New here? Register now
                  </a>
                </p>
              </div>
            </section>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="container"
      >
        <header>
          <p
            class="para"
          >
            Todos
          </p>
        </header>
        <div
          class="container-form"
        >
          <section>
            <h1>
              Sign in
            </h1>
            <div>
              <input
                class="uname"
                id="username"
                placeholder="Username"
                type="email"
                value=""
              />
            </div>
            <br />
            <div>
              <input
                class="pwd"
                id="password"
                placeholder="Password"
                type="password"
                value=""
              />
              <button
                class="btn"
              >
                Sign in
              </button>
            </div>
            <div>
              <p
                class="lnk"
              >
                <a
                  class=""
                  href="/register"
                >
                  New here? Register now
                </a>
              </p>
            </div>
          </section>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`takes an array of objects, each with amount values, & sums the values 1`] = `
<div>
  <div
    class="container"
  >
    <header>
      <p
        class="para"
      >
        Todos
      </p>
    </header>
    <div
      class="container-form"
    >
      <section>
        <h1>
          Sign in
        </h1>
        <div>
          <input
            class="uname"
            id="username"
            placeholder="Username"
            type="email"
            value=""
          />
        </div>
        <br />
        <div>
          <input
            class="pwd"
            id="password"
            placeholder="Password"
            type="password"
            value=""
          />
          <button
            class="btn"
          >
            Sign in
          </button>
        </div>
        <div>
          <p
            class="lnk"
          >
            <a
              class=""
              href="/register"
            >
              New here? Register now
            </a>
          </p>
        </div>
      </section>
    </div>
  </div>
</div>
`;
